ui/apps/akasha/src/extensions/beam-editor/beam-editor.tsx

```diff
@@ -259,7 +273,14 @@ export const BeamEditor: React.FC = () => {
       <Stack customStyle="relative h-full overflow-hidden">
         <Stack customStyle="overflow-auto h-full">
           {blocksInUse.map((block, idx) => (
-            <div key={`${block.key}`} id={`${block.propertyType}-${idx}`}>
+            <div
```

Comment:
Btw the `Card` component has a prop named `type` and a value `plain` which does the same thing as what you have on the div.

Reply:
ok, I'll try it out